infrastructure/ses.tf

```
@@ -0,0 +1,17 @@
+locals {
+  base_domain = "scpca.alexslemonade.org"
+
+  ses_domain = (
+    var.stage == "prod" ? local.base_domain :
+    var.stage == "staging" ? "staging.${local.base_domain}" :
+    "${var.user}.${local.base_domain}"
+  )
```

maybe we can write this as a map to clean up the logic

```suggestion
  ses_domain = locals.domains[var.stage]
```